sdk/java-sdk/src/main/scala/kalix/javasdk/impl/eventsourcedentity/EventSourcedEntitiesImpl.scala

```diff
@@ -51,7 +51,7 @@ final class EventSourcedEntityService(
     val factory: EventSourcedEntityFactory,
     override val descriptor: Descriptors.ServiceDescriptor,
     override val additionalDescriptors: Array[Descriptors.FileDescriptor],
-    val anySupport: AnySupport,
+    val anySupport: MessageCodec,
```

Comment:
`s/anySupport/messageCodec/`

I don't know if we did for the others. If not we can add in this PR as well.

Reply:
yeah, will do that.. 👍

Reply:
solved now